### ISSUE TITLE
Functionality to disable quote escaping

### ISSUE THREAD

**Issue by ksx-kshan** [asafdav/ng-csv#158]
I would like to suggest, or if you would accept a pull-request, the ability to disable the quote escaping via a directive attribute

escape-quotes="false" maybe

I want to be able to output ="123456789" so excel will read this as a formula and not display it in scientific notation

**Comment by dturrina** [comment]
I agree with @ksx-kshan I have a similar issue when trying to force a numeric string to be effectively a string - because of leading zeroes - instead of a number.

**Comment by thinkMX** [comment]
Actually if you need this feature right now and (as me) don't have time to wait until the author accept my pull request... you just have to do these steps:

1.- Open file build/ng-csv.js
2.- Comment the line number 68. It should look like this:
//        data = data.replace(/"/g, '""'); // Escape double qoutes
3.- Enjoy =D
BTW: I needed my accout field to look like this inside the CSV, in order to display correctly without exponential values in Excel:
"=""xxxxxxxxxxxxxxxxxxxx"""
where xxx... is the number
